addons/dango-rain/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Dango rain on profiles (April Fools Day)",
-  "description": "Users that include the word \"dango\" on their \"About Me\" will have dango emojis raining on the sides of their profile.",
+  "description": "Users that include the word \"dango\" on their \"About me\" or \"What I'm working on\" will have dango emojis raining on the sides of their profile.",
```

Comment:
```suggestion
  "description": "Users that include the word \"dango\" on their \"About me\" or \"What I'm working on\" sections will have dango emojis raining on the sides of their profile.",
```

Reply:
What is the reasoning for this change?

Reply:
I don't think most people use these terms like that.

Reply:
> I don't think most people use these terms like that.

People often say things like "comments section". Is there a better word for that here?

Reply:
> > I don't think most people use these terms like that.
> 
> People often say things like "comments section". Is there a better word for that here?

I think they're just standalone... e.g. "I'm going to put that in my WIWO."